BMM.Core/Implementations/DeepLinking/DeepLinkHandler.cs

```diff
@@ -179,7 +179,9 @@ private bool Open(Uri uri, bool shouldLogEvent)
                 return false;
 
             if (shouldLogEvent)
-                _analytics.LogEvent("deep link opened", new Dictionary<string, object> {{"uri", uri.AbsolutePath}});
+                _analytics.LogEvent("deep link opened", new Dictionary<string, object> { { "uri", uri.AbsolutePath } });
```

Comment:
I would change the name of method parameter from "shouldLogEvent" to  isOpenedFromOutsideOfApp. 
Also I wouldn't do if...else here, but simple 

`_analytics.LogEvent(GetDeepLinkOpenedEventName(isOpenedFromOutsideOfApp), new Dictionary<string, object> {{"uri", uri.AbsolutePath}});`

Reply:
refactored